src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
```

Comment:
better to use <link> tag

https://stackoverflow.com/questions/12316501/including-google-fonts-link-or-import